platform/android/MapboxGLAndroidSDK/src/cpp/http_file_source.cpp

```diff
@@ -17,15 +18,20 @@ namespace mbgl {
 
 class HTTPFileSource::Impl {
 public:
-    Impl(const ResourceOptions options): resourceOptions(options.clone()){};
+    Impl(const ResourceOptions resourceOpts, const ClientOptions clientOpts)
```

Comment:
Please use resrouceOptions and clientOptions.

Reply:
Done.